release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -298,8 +298,190 @@ For more information, see xref:../openshift_images/image-streams-manage.adoc#ima
 
 With {product-title} {product-version}, running `oc describe` on an image now returns os/arch and digests of each manifest.
 
-//[id="ocp-4-13-ibm-z"]
-//=== IBM Z and LinuxONE
+[id="ocp-4-13-ibm-z"]
+=== {ibmzProductName} and {linuxoneProductName}
+
+With this release, {ibmzProductName} and {linuxoneProductName} are now compatible with {product-title} {product-version}. The installation can be performed with z/VM or {op-system-base} KVM. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network]
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network]
+
+[discrete]
+==== {ibmzProductName} and {linuxoneProductName} supported new features
+
+The following new features are supported on {ibmzProductName} and {linuxoneProductName} with {product-title} {product-version}:
+
+* Cluster Resource Override Operator
+* Egress IP
+* FIPS cryptography
+* MetalLB Operator
+* Network-Bound Disk Encryption - External Tang Server
+
+[discrete]
+==== IBM Secure Execution
+
+{product-title} now supports configuring {op-system-first} nodes for IBM Secure Execution on {ibmzProductName} and {linuxoneProductName} (s390x architecture).
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.html#installing-rhcos-using-ibm-secure-execution_installing-ibm-z-kvm[Installing {op-system} using IBM Secure Execution]
+
+[discrete]
+==== {ibmzProductName} and {linuxoneProductName} supported features
+
```

Comment:
I suggest a closed lead-in sentence to avoid the "following > following loop". How about:

> {ibmzProductName} and {linuxoneProductName} now support certain {product-title} {product-version} features.